CorePluginFramework/src/au/gov/asd/tac/constellation/pluginframework/logging/DefaultConstellationLogger.java

```diff
@@ -19,44 +19,79 @@
 import au.gov.asd.tac.constellation.pluginframework.Plugin;
 import au.gov.asd.tac.constellation.pluginframework.parameters.PluginParameters;
 import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import org.openide.util.lookup.ServiceProvider;
+import org.openide.windows.TopComponent;
 
 /**
  * A default implementation of a ConstellationLogger that ignores all logging
- * messages. This class exists because the framework expects that the logger is
- * not null and this class provides a no-op implementation that provides a null
- * equivalent.
+ * messages by default. You can set {@link DefaultConstellationLogger#VERBOSE}
+ * to true to enable logging to standard out.
  *
  * @author sirius
+ * @author cygnus_x-1
  */
 @ServiceProvider(service = ConstellationLogger.class, position = Integer.MAX_VALUE)
 public class DefaultConstellationLogger implements ConstellationLogger {
 
+    private static final Logger LOGGER = Logger.getLogger(DefaultConstellationLogger.class.getName());
+
+    private static final boolean VERBOSE = false;
```

Comment:
I'd avoid doing it this way. The better approach would be to set the log level to `Level.FINE` and you activate it by passing  `-J-Dau.gov.asd.tac.constellation.pluginframework.logging.DefaultConstellationLogger.Level=FINE` in `project.properties`. This means that you can enable/disable logging without recompiling.

Reply:
I was considering that - fixed.